packages/fxa-content-server/tests/functional/settings_v2/secondary_email.js

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const intern = require('intern').default;
+const { describe, it, beforeEach } = intern.getPlugin('interface.bdd');
+const selectors = require('../lib/selectors');
+const FunctionalHelpers = require('../lib/helpers');
+const { createEmail } = FunctionalHelpers;
+const FunctionalSettingsHelpers = require('./lib/helpers');
+const { navigateToSettingsV2 } = FunctionalSettingsHelpers;
+
+describe('secondary email', () => {
+  let primaryEmail;
+  const secondaryEmail = createEmail();
+  let click, getEmailCode, testElementExists, testElementTextInclude, type;
```

Comment:
Unfortunately the declaration and initialization of these helper functions must be in two places since `remote` is not available here.